### ISSUE TITLE
Overly enthusiastic copy when user submits a 'no' response to questionnaire

### ISSUE THREAD

**Issue by MyfanwyNixon** [mysociety/writetothem#254]
Fair point from a user:
> "Many thanks - We hope you enjoyed the experience"
> Got the above message from your site after confirming that my MP hadn't got back to me. I didn't have an "experience". Nothing happened.

Looks like it's: https://github.com/mysociety/writetothem/blob/a94678c99f624fe6b704904578c241e419d16b10/templates/website/survey-done.html

Perhaps we just drop "we hope you enjoyed the experience".

**Comment by MyfanwyNixon** [comment]
On a similar note, a user writes:
> « If for some reason we can't send your message or it doesn't get through, we will email you a copy of your message so you can contact your MEPs by more traditional, albeit less fun, means.»

> Do you really believe, I experienced sending my messages as FUN? If so, you must have quite a queer sense of humour.

I reckon this phrase was probably written at WTT's very beginnings, when the web was still a bit of a novelty for many. OTOH I'm quite fond of it, but perhaps worth noting that our relentlessly upbeat copy can grate with some users.

**Comment by MyfanwyNixon** [comment]
The second point has now been fixed with a copy change via https://github.com/mysociety/writetothem/pull/300.

I wasn't 100% confident editing the top issue as I'm not sure of context - is this line seen whether they click 'yes' or 'no'?

**Comment by dracos** [comment]
No, that's the response-yes and response-no templates. That then asks them the second question about "first time", and then they either get survey-done or (1/30 chance) the longer survey (do we still need that survey, @Rebeccarumbul?) which *then* does survey-done.

**Comment by jacksonj04** [comment]
@dracos @Rebeccarumbul: As far as I can remember that survey has been running since before I joined mySociety (assuming it's the demographic one which sends to survey.mysociety.org), and I've never once looked at it other than idle curiosity.

If we want to continue surveying the demographics I would *strongly* recommend we consider migrating it to SurveyGizmo, so we keep all our surveying in one place with a nicer UI. If we don't want this survey any more we should consider removing it, archiving all the data, and looking at turning off survey.mysociety.org (assuming it's not doing anything else, it might be capturing the "useful or not" data from TWFY).

**Comment by dracos** [comment]
I've switched off the survey. This ticket is still open, referring to the 'survey-done' template which is what they get after answering the first time question.

**Comment by MyfanwyNixon** [comment]
>Asking if I enjoyed the service when I have clicked that I got no response whatsoever is a bit of a ridiculous question too!
